src/sass/app/websocket_pages.scss

```diff
@@ -116,6 +116,12 @@
     }
 }
 
+.statementws-content {
+    .container {
+        max-width: 1200px !important;
+    }
+}
+
 #statement-container {
```

Comment:
not sure if we should move this inside line 123, perhaps the diff becomes too large